recipe-client-addon/lib/RecipeRunner.jsm

```diff
@@ -174,13 +199,17 @@ this.RecipeRunner = {
 
       try {
         await manager.runAsyncCallback("postExecution");
+        Uptake.reportAction(actionName, Uptake.ACTION_SUCCESS);
       } catch (err) {
         log.info(`Could not run post-execution hook for ${actionName}:`, err.message);
+        Uptake.reportAction(actionName, Uptake.ACTION_POST_EXECUTION_ERROR);
       }
     }
 
     // Nuke sandboxes
     Object.values(actionSandboxManagers).forEach(manager => manager.removeHold("recipeRunner"));
+
+    Uptake.reportRunner(Uptake.RUNNER_SUCCESS);
```

Comment:
Yes. Runner status is meant to track whether the runner as a whole was successful, vs things that stop the runner entirely (like errors fetching the recipes, or uncaught exceptions). This would allow, for example, us to see what percentage of all recipe runs are affected by a fetch problem, even if, at the same time, an action is also throwing an error.